madara/node/src/service/l2.rs

```diff
@@ -72,6 +72,7 @@ impl Service for SyncService {
         let this = self.start_args.take().expect("Service already started");
         let importer = Arc::new(BlockImporter::new(
             this.db_backend.clone(),
+            // TODO: all_verifications_disabled should be configured from the env
```

Comment:
something I picked from the starkware team, I like this idea of having the TODO with POC mentioned, helps with the resolution really quick!

```suggestion
            // TODO(heemank <insert_date>): all_verifications_disabled should be configured from the env
```

Reply:
Nice, used